test/about/historyTest.js

```diff
@@ -132,14 +136,12 @@ describe('about:history', function () {
     })
     it('selects multiple contiguous rows when shift clicked', function * () {
       yield this.app.client
-        .tabByUrl(aboutHistoryUrl)
-        .loadUrl(aboutHistoryUrl)
         .click('table.sortableTable td.title[data-sort="Brave"]')
         .keys(Brave.keys.SHIFT)
         .click('table.sortableTable td.title[data-sort="https://www.youtube.com"]')
         .waitForVisible('table.sortableTable tr.selected td.title[data-sort="Brave"]')
         .waitForVisible('table.sortableTable tr.selected td.title[data-sort="https://www.youtube.com"]')
-        .waitForVisible('table.sortableTable tr.selected td.title[data-sort="https://brave.com/test"]')
+        .waitForVisible('table.sortableTable td.title[data-sort="https://brave.com/test"]')
```

Comment:
We have sites to test:
- { location: 'https://brave.com', title: 'Brave' },
- { location: 'https://brave.com/test', customTitle: 'customTest' },
- { location: 'https://www.youtube.com' },
- { location: 'https://www.facebook.com' }

Per this test scenario:
- click on [data-sort="Brave"]
- activate shift and click on [data-sort="https://www.youtube.com"]

Selected should be:
- { location: 'https://brave.com', title: 'Brave' }
- { location: 'https://www.youtube.com' }

Not selected should be:
- { location: 'https://brave.com/test', customTitle: 'customTest' }
- { location: 'https://www.facebook.com' }